.github/workflows/workflow.yaml

```diff
@@ -305,11 +304,49 @@ jobs:
         run: |
           make e2e-verify-release IMG=${{ env.IMAGE_REPO }}:${TAG} USE_LOCAL_IMG=false
 
+      - name: Build gator-cli
+        run: |
+          build() {
+            export GOOS="$(echo ${1} | cut -d '-' -f 1)"
+            export GOARCH="$(echo ${1} | cut -d '-' -f 2)"
+            # in case of windows, we need to append .exe to the file name
+            EXTENSION="$(echo ${1} | cut -d '-' -f 3)"
+            EXTENSION=${EXTENSION/exe/.exe}
+            # build the binary
+            make bin/gator-${GOOS}-${GOARCH}${EXTENSION}
+            # rename the binary to gator
+            tmp_dir=$(mktemp -d)
+            cp bin/gator-${GOOS}-${GOARCH}${EXTENSION} ${tmp_dir}/gator${EXTENSION}
+            pushd ${tmp_dir}
+            if [[ ${EXTENSION} == ".exe" ]]; then
+              zip ${GITHUB_WORKSPACE}/_dist/gator-${TAG}-${GOOS}-${GOARCH}.zip gator*.exe
```

Comment:
nit: putting `${GITHUB_WORKSPACE}/_dist/gator-${TAG}-${GOOS}-${GOARCH}` into a would make the differences between these lines more obvious